src/previewFrame/elementHighlighterScript.js

```diff
@@ -0,0 +1,36 @@
+const elementHighlighterScript = `(${function() {
+  window.addEventListener('message', (message) => {
+    const data = JSON.parse(message.data);
+    const {selector: {selector: selectorFromMessage}} = data;
+    const selector = selectorFromMessage;
+    updateCovers(selector);
+  });
+
+  function updateCovers(selector) {
+    const highlighterElements =
+    window.document.querySelectorAll('.popcode_HIGHLIGHTER');
```

Comment:
Completely aesthetic, but I’d be inclined to use a class name like `__popcode-highlighter` to more closely fit the naming convention we use in the main application CSS (with the `__` prefix serving to notate that it’s “internal”)